pac-aurora-provisioner/cloudformation/aurora-pac-master.yml

```diff
@@ -119,9 +123,14 @@ Parameters:
         Type: String
         Default: PAC Aurora
 
+Conditions:
+    UseDBSnapshot: !Not [!Equals [!Ref SourceSnapshot, ""]]
+    DoNotUseDBSnapshot: !Not [Condition: UseDBSnapshot]
```

Comment:
Is this really required? Does `Condition: !Not UseDBSnapshot` not work?

Reply:
No, it doesn't. :S `Condition` value needs to be a `string` that represent a condition. It can not be a condition expression. 💩